### DISCUSSION TITLE
SMTP 5xx error: permanent error (550)

### DISCUSSION THREAD

**Discussion by ftribrunn** [dani-garcia/vaultwarden#2383] [Q&A] [answered]
I got the following Error:

```
][request][INFO] POST /api/accounts/register
[2022-03-21 16:02:39.210][vaultwarden::mail][ERROR] SMTP 5xx error: permanent error (550)
[2022-03-21 16:02:39.211][vaultwarden::api::core::accounts][ERROR] Error sending welcome email: SMTP 5xx error: permanent error (550)

```
but sending testmail works fine

```
[2022-03-21 15:47:27.067][request][INFO] POST /admin/test/smtp/
[2022-03-21 15:47:27.830][response][INFO] POST /admin/test/smtp (test_smtp) => 200 OK

```

I have no idea...any suggestions?
thx in advance

**Comment by BlackDex** [comment, marked as answer]
Are you sending that test mail to the exact same address as the register mail?
If not, it could be the recipient.
When yes, then it could be the contents of the mail which is somehow blocked by the SMTP server.

You could try to enable SMTP_DEBUG=true for a short while and see if you get a better description back from the mail server.
Else i suggest to reach out to the SMTP mail provider and see if they can help  you.

**Reply by ftribrunn** [reply]
Hi!
Thanks a lot! I tried to send the Tesmail on the same email - and it does not work. so I look deeper and found a dns problem on the server for that specific domain.
so its fixed and working.
my vault - thx for helping me out :-)

**Reply by frostbox42** [reply]
For anyone else finding their way to this post with the same problem.
I had the same symptoms, and the cause was that I was using AWS SES to send out emails, and hadn't authenticated the email/domain I was sending to, hence the emails weren't.

**Comment by brockdarnold** [comment]
I came to this thread with a similar issue. The fix was using an app generated password through google, and changing the smtp host from smtp-relay.gmail.com to smtp.gmail.com since I was not a google organization and was following the wrong instruction pages from google.
